changes/620.feature.rst

```diff
@@ -0,0 +1 @@
+Add source_xpos, source_ypos, and meta.cal_step.ta_center attributes to core schema in support of new ta_center step
```

Comment:
Would you update this changelog now that source_xpos and source_ypos aren't in the core schema (as you noted https://github.com/spacetelescope/stdatamodels/pull/620/files#r2550969082)?